mappings/net/minecraft/world/dimension/DimensionType.mapping

```diff
@@ -11,9 +11,18 @@ CLASS net/minecraft/class_2874 net/minecraft/world/dimension/DimensionType
 	METHOD <init> (Ljava/util/OptionalLong;ZZZZDZZIIILnet/minecraft/class_6862;Lnet/minecraft/class_2960;FLnet/minecraft/class_2874$class_7512;)V
 		ARG 1 fixedTime
 		ARG 2 hasSkylight
+		ARG 3 hasCeiling
 		ARG 4 ultrawarm
+		ARG 5 natural
 		ARG 6 coordinateScale
-		ARG 9 piglinSafe
+		ARG 7 bedWorks
```

Comment:
`coordinateScale` is a double, it takes two slots. Please use enimga to map stuff.